server/repositories/cmsQueries/audioPageQuery.js

```diff
@@ -58,6 +58,11 @@ class AudioPageQuery {
     }));
 
   transform(item) {
+    const url =
+      item.fieldMojThumbnailImage?.imageStyleUri
+        ?.map(image => image?.tile_large)
+        .find(image => Boolean(image)) || item.fieldMojThumbnailImage?.uri?.url;
+
```

Comment:
I feel we should be making some utils here. We seem to be doing the same extractions for all content 🤔